client/src/main/java/io/confluent/kafka/schemaregistry/client/CachedSchemaRegistryClient.java

```diff
@@ -31,14 +34,19 @@
 
 public class CachedSchemaRegistryClient implements SchemaRegistryClient {
 
-  private final String baseUrl;
+  private UrlList baseUrls;
   private final int identityMapCapacity;
   private final Map<String, Map<Schema, Integer>> schemaCache;
   private final Map<Integer, Schema> idCache;
   private final Map<String, Map<Schema, Integer>> versionCache;
 
-  public CachedSchemaRegistryClient(String baseUrl, int identityMapCapacity) {
-    this.baseUrl = baseUrl;
+  public static CachedSchemaRegistryClient newInstance(String baseUrlConfig, int identityMapCapacity) {
```

Comment:
Ah, that's a fine policy. We haven't really standardized anything like that, it's fine as is. Just curious.